components/camel-platform-http/src/main/java/org/apache/camel/component/platform/http/spi/Method.java

```diff
@@ -16,11 +16,7 @@
  */
 package org.apache.camel.component.platform.http.spi;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Locale;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
We do not use wildcard imports

Reply:
Fixed. It's IDE's optimization

Reply:
Sorry for accidentally merging branch apache:main in